embedded/sql/engine_test.go

```diff
@@ -5903,42 +5903,43 @@ func setupCommonTestWithOptions(t *testing.T, sopts *store.Options) (*Engine, *s
 	require.NoError(t, err)
 	t.Cleanup(func() { closeStore(t, st) })
 
-	ctx := context.TODO()
 	engine, err := NewEngine(st, DefaultOptions().WithPrefix(sqlPrefix))
 	require.NoError(t, err)
 
-	_, _, err = engine.Exec(ctx, nil, "CREATE DATABASE db1;", nil)
+	_, _, err = engine.Exec(context.Background(), nil, "CREATE DATABASE db1;", nil)
```

Comment:
do we need a new context every call? Why can't we declare a variable like before and use it throughout?

Reply:
yes, it's not needed, I did it so to have total control of the control on each call, emulating the most typical manner in which is used. It could be as before as well